pencil/README.md

```diff
@@ -0,0 +1,59 @@
+![Pencil](https://cldup.com/63hb_S0bDS.png)
```

Comment:
The "documentation" in this repo is usually represented by comments in the Dockerfiles (which you already did) This readme may be redundant and cause a small "maintenance overhead" don't you think?